### PR TITLE
update to v1 apis and zapx versions

### PR DESCRIPTION
**Pull request by mschoch**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,17 +4,17 @@ go 1.13
...
 require (
 	github.com/RoaringBitmap/roaring v0.4.23
-	github.com/blevesearch/bleve_index_api v0.0.8
+	github.com/blevesearch/bleve_index_api v1.0.0
 	github.com/blevesearch/go-porterstemmer v1.0.3
-	github.com/blevesearch/scorch_segment_api v0.0.10
+	github.com/blevesearch/scorch_segment_api v1.0.0
 	github.com/blevesearch/segment v0.9.0
 	github.com/blevesearch/snowballstem v0.9.0
 	github.com/blevesearch/upsidedown_store_api v0.0.1
```
Comment (abhinavdangeti) [inline go.mod:12]: The README in https://github.com/blevesearch/upsidedown_store_api needs to be updated, now that it'll be officially used. Think that needs to be done before this change here?

Comment (mschoch) [reply go.mod:12]: oh, well ya that should have been before i tagged v1.0.0.  so much for the checklist...

**Comment by mschoch** [comment]
also appears my fix to the integration test doesn't work, as we see it still sometimes returns results in the wrong order

File: go.mod (hunk 1 above)
Comment (abhinavdangeti) [reply go.mod:12]: Heh, just spotted it.

**Review by abhinavdangeti — APPROVED** [review]

**Review by abhinavdangeti — APPROVED** [review]

**Pull request merged**